core/services/chainlink/testdata/config-full.toml

```diff
@@ -0,0 +1,275 @@
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
+
+[Database]
+ListenerMaxReconnectDuration = '1m0s'
+ListenerMinReconnectInterval = '5m0s'
+Migrate = true
+ORMMaxIdleConns = 7
+ORMMaxOpenConns = 13
+TriggerFallbackDBPollInterval = '2m0s'
+AdvisoryLockCheckInterval = '5m0s'
+AdvisoryLockID = 345982730592843
+LockingMode = 'advisory'
+BackupDir = 'test/backup/dir'
+BackupOnVersionUpgrade = true
+BackupURL = 'http://test.back.up/fake'
+
+[TelemetryIngress]
+UniConn = true
+Logging = true
+ServerPubKey = 'test-pub-key'
+URL = 'https://prom.test'
+BufferSize = 1234
+MaxBatchSize = 4321
+SendInterval = '1m0s'
+SendTimeout = '5s'
+UseBatchSend = true
+
+[Log]
+JSONConsole = true
+FileDir = 'log/file/dir'
+SQL = true
+FileMaxSize = '100.00gb'
+FileMaxAgeDays = 17
+FileMaxBackups = 9
+UnixTS = true
+
+[WebServer]
+AllowOrigins = '*'
+AuthenticatedRateLimit = 42
+AuthenticatedRateLimitPeriod = '1s'
+BridgeResponseURL = 'https://bridge.response'
+HTTPWriteTimeout = '1m0s'
+Port = 56
+SecureCookies = true
+SessionTimeout = '1h0m0s'
+UnAuthenticatedRateLimit = 7
+UnAuthenticatedRateLimitPeriod = '1m0s'
+RPID = 'test-rpid'
+RPOrigin = 'test-rp-origin'
+TLSCertPath = 'tls/cert/path'
+TLSHost = 'tls-host'
+TLSKeyPath = 'tls/key/path'
+TLSPort = 6789
+TLSRedirect = true
+
+[JobPipeline]
+DefaultHTTPLimit = 67
+DefaultHTTPTimeout = '1m0s'
+FeatureExternalInitiators = true
+MaxRunDuration = '1h0m0s'
+ReaperInterval = '4h0m0s'
+ReaperThreshold = '168h0m0s'
+ResultWriteQueueDepth = 10
+
+[OCR2]
+ContractConfirmations = 11
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+ContractTransmitterTransmitTimeout = '1m0s'
+DatabaseTimeout = '8s'
+KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000'
+MonitoringEndpoint = 'test-mon-end'
+
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+DefaultTransactionQueueDepth = 12
+KeyBundleID = 'acdd42797a8b921b2910497badc5000600000000000000000000000000000000'
+MonitoringEndpoint = 'test-monitor'
+SimulateTransactions = true
+TransmitterAddress = '0xa0788FC17B1dEe36f057c42B6F373A34B014687e'
+OutgoingMessageBufferSize = 7
+IncomingMessageBufferSize = 3
+DHTLookupInterval = 9
+BootstrapCheckInterval = '1m0s'
+NewStreamTimeout = '1s'
+
+[P2P]
+NetworkingStack = 'V1V2'
+IncomingMessageBufferSize = 13
+OutgoingMessageBufferSize = 17
+AnnounceIP = '1.2.3.4'
+AnnouncePort = 1234
+BootstrapCheckInterval = '1m0s'
+BootstrapPeers = ['foo', 'bar', 'should', 'these', 'be', 'typed']
+DHTAnnouncementCounterUserPrefix = 4321
+DHTLookupInterval = 9
+ListenIP = '4.3.2.1'
+ListenPort = 9
+NewStreamTimeout = '1s'
+PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw'
+PeerstoreWriteInterval = '1m0s'
+V2AnnounceAddresses = ['a', 'b', 'c']
+V2Bootstrappers = ['1', '2', '3']
+V2DeltaDial = '1m0s'
+V2DeltaReconcile = '1s'
+V2ListenAddresses = ['foo', 'bar']
```

Comment:
```suggestion
[P2P]
IncomingMessageBufferSize = 13
OutgoingMessageBufferSize = 17

[P2P.V1]
AnnounceIP = '1.2.3.4'
AnnouncePort = 1234
BootstrapCheckInterval = '1m0s'
BootstrapPeers = ['foo', 'bar', 'should', 'these', 'be', 'typed']
DHTAnnouncementCounterUserPrefix = 4321
DHTLookupInterval = 9
ListenIP = '4.3.2.1'
ListenPort = 9
NewStreamTimeout = '1s'
PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw'
PeerstoreWriteInterval = '1m0s'

[P2P.V2]
AnnounceAddresses = ['a', 'b', 'c']
Bootstrappers = ['1', '2', '3']
DeltaDial = '1m0s'
DeltaReconcile = '1s'
ListenAddresses = ['foo', 'bar']
```
@samsondav In this case, we could even remove the explicit `NetworkingStack` field.

Reply:
Done